crate_universe/src/rendering.rs

```diff
@@ -28,13 +28,13 @@ use crate::utils::{self, sanitize_repository_name};
 type Platforms = BTreeMap<String, BTreeSet<String>>;
 
 pub struct Renderer {
-    config: RenderConfig,
+    config: Config,
     engine: TemplateEngine,
 }
 
 impl Renderer {
-    pub fn new(config: RenderConfig) -> Self {
-        let engine = TemplateEngine::new(&config);
+    pub fn new(config: Config) -> Self {
```

Comment:
Why not have this continue to take a `RenderConfig` and additionally take the list of supported platforms? This way if `Config` grows the impact is more contained?

Reply:
Works for me 🙂